docs/formats.rst

```diff
@@ -231,36 +231,36 @@ is one of many standards in this area.
 
 XLIFF is usually used as bilingual, but Weblate supports it as monolingual as well.
 
-Translations states
+Translation states
 +++++++++++++++++++
 
 .. versionchanged:: 3.3
 
-   Weblate did ignore the state attribute prior to the 3.3 release.
+   Weblate ignored the state attribute prior to the 3.3 release.
 
-The ``state`` attribute in the file is partially processed and mapped to needs
-edit state in Weblate (the following states are used to flag the string as
-needing edit if there is some target present: ``new``, ``needs-translation``,
+The ``state`` attribute in the file is partially processed and mapped to the
+"Needs edit" state in Weblate (the following states are used to flag the string as
```

Comment:
I feel like this has been changed to "Marked for edit" in itself not ideal, but I can't find out what "Needs edit" is now.

Reply:
Not change, see https://github.com/WeblateOrg/weblate/issues/2240